extensions/adobe/experience/profile/profile-all.schema.json

```diff
@@ -28,6 +28,44 @@
           "additionalProperties": {
             "$ref": "https://ns.adobe.com/xdm/context/namespace"
           }
+        },
+        "xdm:associatedIdentities": {
```

Comment:
Field name should be https://ns.adobe.com/experience/profile/associatedIdentities